docker/docker-compose.1604.52.yaml

```diff
@@ -33,6 +33,8 @@ services:
       - MAX_ALLOCS_ALLOWED_modifying_byte_buffer_view=2010 # 5.2 improvement 4000
       - MAX_ALLOCS_ALLOWED_schedule_10000_tasks=90050
       - MAX_ALLOCS_ALLOWED_read_10000_chunks_from_file=210050
+      - MAX_ALLOCS_ALLOWED_udp_1000_reqs_1_conn=16211
+      - MAX_ALLOCS_ALLOWED_udp_1_reqs_1000_conn=226000
```

Comment:
Oh, do we understand why 5.2 is worse than 5.1? Seems to be 3 allocations worse in 5.2, we should analyse why that is and either fix it on our side or file a Swift bug. Newer Swifts should only make it better :).

Reply:
Instability as noted below could be the cause.  Could well be a swift bug but that's a separate issue.